Dockerfile

```diff
@@ -1,15 +1,22 @@
-FROM node:14
+FROM docker.io/node:14
```

Comment:
What's the difference between `node:14` and `docker.io/node:14`? I thought docker.io was the default? Does it need to be explicitly defined in order to build properly on GitHub? :thinking:

Reply:
They're the same, docker.io is the default for docker tooling. We don't _need_ to specify it for local or autobuilds but it's a Good Thing to get into the habit of doing. Defaults might change, we could switch to buildah, etc.